docs/content/specs-defs/includes/bicep/resource/non-functional/BCPRMNFR3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: BCPRMNFR3 - Resource module structure
+url: /spec/BCPRMNFR3
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+type: posts
+tags: [
+  Class-Resource, # MULTIPLE VALUES: this can be "Class-Resource" AND/OR "Class-Pattern" AND/OR "Class-Utility"
+  Type-NonFunctional, # SINGLE VALUE: this can be "Type-Functional" OR "Type-NonFunctional"
+  Category-Naming/Composition, # SINGLE VALUE: this can be "Category-Testing" OR "Category-Telemetry" OR "Category-Contribution/Support" OR "Category-Documentation" OR "Category-CodeStyle" OR "Category-Naming/Composition" OR "Category-Inputs/Outputs" OR "Category-Release/Publishing"
+  Language-Bicep, # MULTIPLE VALUES: this can be "Language-Bicep" AND/OR "Language-Terraform"
+  Severity-MUST, # SINGLE VALUE: this can be "Severity-MUST" OR "Severity-SHOULD" OR "Severity-MAY"
+  Persona-Owner, # MULTIPLE VALUES: this can be "Persona-Owner" AND/OR "Persona-Contributor"
+  Persona-Contributor, # MULTIPLE VALUES: this can be "Persona-Owner" AND/OR "Persona-Contributor"
+  Lifecycle-BAU, # SINGLE VALUE: this can be "Lifecycle-Initial" OR "Lifecycle-BAU" OR "Lifecycle-EOL"
+  Validation-TBD # SINGLE VALUE: this can be "Validation-BCP/Manual" OR "Validation-BCP/CI/Informational" OR "Validation-BCP/CI/Enforced"
+]
+priority: 13010
+---
+
+#### ID: BCPRMNFR3 - Implementing child resources
```

Comment:
Not sure if that was discussed: is this line supposed to be the same as the title? Why having them different if they have the same ID?

Reply:
We talked about it. It does not need to be - the one on top is just the one shown in the table.
If I recall correctly, @matebarabas said that this title here can be longer.
Anyways - I agree that in this case they should be more aligned. I think it made more sense when the category was still around, but as we removed it ... 
Will work on the naming

Reply:
done